src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Shared/MessageBox.razor.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace TodoTemplate.App.Shared;
+
+public partial class MessageBox : IDisposable
+{
+    private static event Action<string, string> OnShow = default!;
+
+    public static void Show(string message, string title = "")
+    {
+        OnShow?.Invoke(message, title);
+    }
+
+    protected override void OnInitialized()
+    {
+        MessageBox.OnShow += ShowMessageBox;
+
+        base.OnInitialized();
+    }
+
+    private void ShowMessageBox(string message, string title)
+    {
+        InvokeAsync(() =>
+        {
+            IsOpen = true;
+
+            Title = title;
+            Body = message;
+
+            StateHasChanged();
+        });
+    }
+
+    // ========================================================================
+
+    private bool IsOpen { get; set; }
```

Comment:
Is the property definition of private correct? It would not be better to define a field instead of a property.

Reply:
it's a convention of the Bit Open Source projects 😁